test/data/echopulse/pcschema.xml

```diff
@@ -3,66 +3,66 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
I don't understand why we need still need the `pcschema.xml` file in the `test/data/echopulse` directory.